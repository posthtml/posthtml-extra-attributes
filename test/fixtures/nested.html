<div>
  Parent
  <span>Child</span>
</div>
