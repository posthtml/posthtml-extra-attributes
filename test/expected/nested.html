<div class="parent">
  Parent
  <span class="child">Child</span>
</div>
